api/repositories/service_binding_repository_test.go

```diff
@@ -543,40 +554,61 @@ var _ = Describe("ServiceBindingRepo", func() {
 				).To(Succeed())
 
 				Expect(serviceBinding.Labels).To(HaveKeyWithValue("servicebinding.io/provisioned-service", "true"))
-				Expect(serviceBinding.Spec).To(Equal(
-					korifiv1alpha1.CFServiceBindingSpec{
-						DisplayName: nil,
-						Service: corev1.ObjectReference{
-							Kind:       "CFServiceInstance",
-							APIVersion: korifiv1alpha1.SchemeGroupVersion.Identifier(),
-							Name:       cfServiceInstance.Name,
-						},
-						AppRef: corev1.LocalObjectReference{
-							Name: appGUID,
-						},
-					},
-				))
+				Expect(serviceBinding.Spec.Type).To(Equal(korifiv1alpha1.CFServiceBindingTypeApp))
```

Comment:
Checking the spec's field one by one with `Equal` is a bit awkward. What you have done is perfectly fine, but if you want to, you could only match the fields that are interesting to this test. For example, the code does not set the `Spec.Service.APIVersion` explicitly, so no need to check it

Here is an [example](https://github.com/cloudfoundry/korifi/blob/cf08b4aa29b0cd06af66fe72b0ad813d46ea65e0/api/repositories/service_broker_repository_test.go#L294) of `MatchFields` where we only check the fields we are interested in.